src/molecule_podman/playbooks/create.yml

```diff
@@ -178,6 +178,7 @@
       changed_when: true
 
     - name: Wait for instance(s) creation to complete
+      become: "{{ not (item.item.rootless|default(true)) }}"
```

Comment:
Have you tried `default(omit)` here? I think it would be better than relying on true as that would no longer respect command line arguments related to become. Omit looks more sensitive to me.

Reply:
Indeed that's much better, it's even more readable, i didn't know this filter, thank you for the tip, il works this way too !